R/demographics-table.R

```diff
@@ -0,0 +1,80 @@
+
+
+smr <- function(value = seq(1,5)) {
+  tibble(
+    `mean (sd)` = paste0(sig(mean(value, na.rm = TRUE)), " (", sig(sd(value,na.rm=TRUE)), ")"),
+    `min-max` = paste0(sig(range(value,na.rm=TRUE)), collapse = " - ")
+  )
+}
+
+#' This is the title
+#'
+#' This function makes a single table from both continuous and categorical data.
+#'
+#' @param data
+#' @param cols_cont
+#' @param cols_cat
+#'
+#' @return
+#' An object of class `pmtable`
+#'
+#' @examples
+#'
+#' @seealso [pt_cont_long()], [pt_cat_long()]
+#'
+#' @export
+pt_demographics <- function(data, cols_cont, cols_cat, span, units = NULL,
```

Comment:
Let's let the user pass a function; rename that `smr()` function to something in line with what we do for say `pt_cont_long` and use it as the default.

Reply:
@kylebaron Oops just saw this. Ill make the change back to the original code then